src/platform/packages/shared/kbn-scout/src/playwright/fixtures/scope/test/scout_page/index.ts

```diff
@@ -29,9 +29,13 @@ export type ScoutPage = Page & {
   gotoApp: (appName: string, pathOptions?: PathOptions) => ReturnType<Page['goto']>;
   /**
    * Waits for the Kibana loading spinner indicator to disappear.
+   * @param options - Optional configuration object.
+   * @param options.timeout - The timeout in milliseconds to wait for the indicator to disappear (default: 10000).
    * @returns A Promise resolving when the indicator is hidden.
    */
-  waitForLoadingIndicatorHidden: () => ReturnType<Page['waitForSelector']>;
+  waitForLoadingIndicatorHidden: (options?: {
+    timeout?: number;
+  }) => ReturnType<Page['waitForSelector']>;
```

Comment:
Unfortunately our CI stats show that global loading indicator is a source of flakiness and can't be used to confirm if particular UI component rendering is completed.

We might have to remove it from helpers to avoid confusion, so the question is if you can wait for something more specific in each page you test?